modules/refined-tests/src/test/scala/ru/m2/calypso/refined/RefinedKeysSuite.scala

```diff
@@ -0,0 +1,13 @@
+package ru.m2.calypso.refined
+
+import eu.timepit.refined.scalacheck.string._
+import eu.timepit.refined.types.string.NonEmptyString
+import org.scalatest.funsuite.AnyFunSuiteLike
+import org.scalatest.prop.Configuration
+import org.typelevel.discipline.scalatest.FunSuiteDiscipline
+import ru.m2.calypso.refined.MissingInstances._
+import ru.m2.calypso.testing.KeyCodecTests
+
+class RefinedKeysSuite extends AnyFunSuiteLike with FunSuiteDiscipline with Configuration {
+  checkAll("KeyCodec[NonEmptyString]", KeyCodecTests[NonEmptyString].codec)
```

Comment:
I've noticed we have `KeyEncoder[Int]`, but pretty much no build-in support for refined versions of it - no instances in `ru.m2.calypso.refined.MissingInstances`, no tests. Was just curious: is it intentional?

I could see why that would be the case as there are quite a few types we would maybe want to cover such as `NonNeg` and `PosInt`, which are perhaps not the most commonly used. Should we even have `KeyEncoder[Int]` in the first place then? I'm not sure.

Just something to think about if you haven't already.

Reply:
Yes, there was no need yet. When someone asks, we will add it.